docs/index.md

```diff
@@ -34,6 +34,15 @@ agile, flexible, rapid and secure manner. [more...](rocm)
 :::
 ::::
 
+::::{grid-item}
+:::{dropdown} [Announcements](announcements)
+
+- [Planned Deprecations](deprecations)
+- [Docs-As-Code](docs-as-code)
```

Comment:
is the plan to have separate pages for announcements and docs-as-code?

Reply:
These were just sample announcements, could be anything here, maybe limit 5 and rest as an archive.